core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.spec.ts

```diff
@@ -113,7 +113,7 @@ describe('VisualizationPanelContentComponent', () => {
     const testData: Record<string, ResultObject> = {
       'operator1': {
         operatorID: 'operator1', chartType: ChartType.HTML_VIZ,
-        table: [{'HTML-content': '<div>sample</div>'}], totalRowCount: 1
+        table: [{'html-content': '<div>sample</div>'}], totalRowCount: 1
```

Comment:
the test case should also be changed to lowercase.

Reply:
it's already changed.